bootstrap/bootstrap.go

```diff
@@ -1476,7 +1474,8 @@ func (b *Bootstrap) runPreCommandHooks(ctx context.Context) error {
 
 // runCommand runs the command and adds tracing spans.
 func (b *Bootstrap) runCommand(ctx context.Context) error {
-	span, ctx := tracetools.StartSpanFromContext(ctx, "command", b.Config.TracingBackend)
+	spanName := b.implementationSpecificSpanName("command-exec", "command")
+	span, ctx := tracetools.StartSpanFromContext(ctx, spanName, b.Config.TracingBackend)
```

Comment:
Hrm, is this span even necessary?

In the switch statement below, it'll either create a `plugin command hook` or a `repository command hook` or an `agent command hook`, _or_ it will run the `defaultCommandPhase()`. AFAIK only that final case (which is the 99% default) is lacking a span.

I wonder if we should replace this `command-exec` span with a `default command hook` span just for that final/default case?

Reply:
yep, sounds good!

Reply:
oh wait, actually, we don't even need to do that, there's already a span for the default command hook. I can't remember why i added this, ahaha